templates/etc/flapjack/flapjack_config.erb

```diff
@@ -283,7 +283,14 @@ production:
       port: <%=@prd_web_ui_port%>
       timeout: <%=@prd_web_ui_timeout%>
       access_log: '<%=@prd_log_dir-%><%=@prd_web_ui_accesslog-%>'
+<% if @prd_web_ui_base_url -%>
+      base_url: '<%=@prd_web_ui_base_url-%>'
+<% end -%>
```

Comment:
This might be better expressed as:
```erb
<% if @prd_web_ui_base_url %>
<%   if @prd_web_ui_port %>
  base_url: '<%= @prd_web_ui_base_url -%>:<%= @prd_web_ui_base_port -%>'
<%   else %>
  base_url: '<%=@prd_web_ui_base_url -%>'
<%   end %>
<% end %>
```
with similar behaviour for the other URL-handling sections. (Not sure what indenting is appropriate there, adjust as required.)

It's actually probably better that these be handled in Flapjack code itself, but that'll be as part of a major version change/interface breakage I'd think.